includes/modules/sideboxes/featured_categories.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * featured sidebox - displays a random Featured Catelog
+ *
+ * @copyright Copyright 2003-2022 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: DrByte 2020 Jul 10 Modified in v1.5.8-alpha $
+ * based on featured and brands
+ */
+
+// test if box should display
+  $show_featured= true;
+
+  if ($show_featured == true) {
+    $random_featured_categories_query = "select p.categories_id, p.categories_image, pd.categories_name
+                           from (" . TABLE_CATEGORIES . " p
+                           left join " . TABLE_FEATURED_CATEGORIES . " f on p.categories_id = f.categories_id
+                           left join " . TABLE_CATEGORIES_DESCRIPTION . " pd on p.categories_id = pd.categories_id )
+                           where p.categories_id = f.categories_id
+                           and p.categories_id = pd.categories_id
```

Comment:
- Use `c` and `cd` as the aliases instead of `p` and `pd`.
- Uppercase SQL keywords, e.g. `select` -> 'SELECT'
- No quotes around the `language_id` clause value, it's an integer.

Reply:
Still need to remove quotes around the `language_id` clause value.